src/utils/eip712.ts

```diff
@@ -0,0 +1,97 @@
+import { utils } from "ethers";
+import { EIP712TypedData, Maker } from "../types";
+
+// EIP 712 (Typed structured data hashing and signing) related data
+// https://eips.ethereum.org/EIPS/eip-712
+
+export const MAKER_HASH = "0x003c1bce41a2de73dfe64d6eeb2b3d7f15f1c0c382d9d963c2c6daeb75f0e539";
```

Comment:
As a constant, should that be in `src/constants/eip712.ts`?

Reply:
it's only used in this file tho, I'm even considering removing the export